circuit_puzzles/programs/announcer_configure.clsp

```diff
@@ -0,0 +1,88 @@
+; governance backdoor
+; we either activate this announcer or deactivate, can happen at any time via CRT vote
+(mod ((MOD_HASH STATUTES_STRUCT LAUNCHER_ID INNER_PUZZLE_HASH APPROVED DEPOSIT DELAY ATOM_VALUE COOLDOWN_START TIMESTAMP_EXPIRES)
+      ; solution
+      inner_puzzle_hash new_puzzle_hash deposit input_conditions
+      (current_timestamp statutes_inner_puzzle_hash input_toggle_activation deactivation_cooldown_interval delay new_price melt)
+     )
+  (include condition_codes.clib)
+  (include curry.clib)
+  (include *standard-cl-23*)
+  (include utils.clib)
+  (include statutes_utils.clib)
+  (include announcer.clib)
+
+  (assign
+    statutes_puzzle_hash (calculate-statutes-puzzle-hash STATUTES_STRUCT statutes_inner_puzzle_hash)
+    (toggle_activation . cooldown_start) (
+      if
+        (any (= input_toggle_activation 0) (= input_toggle_activation 1))
+        (if
+          (= input_toggle_activation 0)
+          (if
+            (all
+              APPROVED
+              (> COOLDOWN_START 0)
+              (> (- current_timestamp COOLDOWN_START) deactivation_cooldown_interval)
+            )
+            ; deactivating announcer
+            (c 0 0)
+            (if (= COOLDOWN_START 0)
+              ; starting the cooldown
+              (c 1 (+ deactivation_cooldown_interval current_timestamp))
+              ; keep the current state
+              (c 1 COOLDOWN_START)
+            )
+          )
+          (if (= COOLDOWN_START 0)
+            ; keep the current state
+            (c APPROVED COOLDOWN_START)
+            ; disable the cooldown and keep the announcer approved
+            (c APPROVED 0)
+          )
+        )
+        (x)
+      )
+    operation_conditions (if melt
+      (if (not APPROVED)
+        ; melting into normal XCH coin
+        (list
+          (list CREATE_COIN INNER_PUZZLE_HASH DEPOSIT (list INNER_PUZZLE_HASH))
+        )
+        (x) ; can't melt approved announcer
+      )
+      (recreate-myself-condition MOD_HASH
+        STATUTES_STRUCT
+        LAUNCHER_ID
+        (if new_puzzle_hash new_puzzle_hash INNER_PUZZLE_HASH)
+        toggle_activation ; current statutes price counter passed
+        deposit
+        delay
+        (if new_price new_price ATOM_VALUE)
+        cooldown_start
+        (+ current_timestamp delay)
+        input_conditions
+      )
+    )
+    (assert
+      inner_puzzle_hash  ; must be owner
+      (li
+        (assert-statute statutes_puzzle_hash STATUTE_ANNOUNCER_DELAY delay)
+        (assert-statute statutes_puzzle_hash STATUTE_ANNOUNCER_MIN_DEPOSIT deposit)
```

Comment:
do we want deposit to equal Statutes min deposit, or do we want to allow deposit to be any value greater than Statutes min deposit?

Reply:
equal to, need to rename it